config/initializers/active_record_extensions.rb

```diff
@@ -8,11 +8,11 @@ class ActiveRecord::Base
 # Permit safemode template rendering to have basic read-only access over
 # model relations
 class ActiveRecord::AssociationRelation::Jail < Safemode::Jail
-  allow :[], :each, :first, :to_a
+  allow :[], :each, :first, :to_a, :map
```

Comment:
Is this related to the issue? I think this could be a very dangerous addition - this about something like `host.interfaces.map(&:destroy)`